packages/shared/src/modules/groups/libs/validation-schemas/create-group.validation-schema.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from "zod";
+
+import { GroupValidationMessage, GroupValidationRule } from "../enums/enums.js";
+import { type GroupCreateRequestDto } from "../types/types.js";
+
+const groupCreate: z.ZodType<GroupCreateRequestDto> = z.object({
+	name: z.string().max(GroupValidationRule.NAME_MAXIMUM_LENGTH, {
+		message: GroupValidationMessage.NAME_TOO_LONG,
+	}),
+	permissionIds: z.array(z.number().int().positive()),
+	userIds: z.array(z.number().int().positive()),
```

Comment:
Will there be no error messages?

Reply:
Oh... I was planning to add them later, but I forgot